Samples/SensorActivityMonitorConsoleApp/SensorActivityMonitorConsoleApp/main.cpp

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (C) Microsoft Corporation. All rights reserved.
+
+#include "pch.h"
+
+using namespace winrt::Windows::Devices::Enumeration;
+
+HRESULT IsCameraInUse(
+    _In_z_ LPCWSTR symbolicName,
+    bool& inUse
+)
+{
+    winrt::com_ptr<MyCameraNotificationCallback> cameraNotificationCallback;
```

Comment:
>com_ptr [](http://example.com/codeflow?start=11&length=7)

why not wil::com_ptr?

Reply:
does it need to be? (is it projected? or derived from IInspectable?) .  
I am only trying to see if we have less confusions for the developer reading the sample.

Reply:
It's just classic COM done by winrt by including unknwn.h or wil\cppwinrt.h before winrt headers as described here https://docs.microsoft.com/en-us/windows/uwp/cpp-and-winrt-apis/author-coclasses#enabling-classic-com-support